policyengine_us/variables/household/income/spm_unit/deep_poverty_gap.py

```diff
@@ -12,6 +12,6 @@ class deep_poverty_gap(Variable):
     definition_period = YEAR
 
     def formula(spm_unit, period, parameters):
-        income = spm_unit("spm_unit_net_income", period)
+        resources = spm_unit("spm_resources", period)
         deep_poverty_threshold = spm_unit("deep_poverty_line", period)
         return max_(deep_poverty_threshold - income, 0)
```

Comment:
```suggestion
        return max_(deep_poverty_threshold - resources, 0)
```

Reply:
the same change is needed everywhere, please go through all the formulas that you changed and check whether the variables that you are using still exist 
If you rename income to resources then you will have to change it in other places of the formula